docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.9"
+
+services:
+
+  as:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - "8000:8000"
+    extra_hosts:
+      - "hostnet:192.168.112.1"
+    volumes:
+      - "./example/:/run/spid-django/example/"
+      - "./src/:/run/spid-django/src/"
+
+  spid-testenv2:
+    image: italia/spid-testenv2:1.1.0
+    ports:
+      - "8088:8088"
+    extra_hosts:
+      - "hostnet:192.168.112.1"
+    volumes:
+      - "./example/configs/spid-testenv2/config.yaml:/app/conf/config.yaml:ro"
+      - "./example/configs/spid-testenv2/users.json:/app/conf/users.json:ro"
+
+  spid-saml-check:
+    build:
+      context: ./spid-saml-check/
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    extra_hosts:
+      - "hostnet:192.168.112.1"
+    volumes:
+      - "./example/configs/spid-saml-check/idp.json:/spid-saml-check/spid-validator/config/idp.json:ro"
+
+networks:
+  default:
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+        - subnet: 192.168.112.0/20
```

Comment:
anche qui

Reply:
si può mettere anche qui, ma non so quanto possa essere utile: questo pezzo serve solamente per avere un indirizzo gateway fisso e non generato casualmente da docker quando al momento della creazione del default network per la prima volta.

Anche in questo caso quali e quante variabili vuoi?

Reply:
fai un esempio di compose env e rimanda alla doc ufficiale di Docker per opportune integrazioni, e via, al chiudiamo così

Reply:
Ho aggiunto il file `docker-compose.env` e aggiornato il README.md per utilizzarlo. Risolvo la conversazione ora "Outdated", se c'è altro segnalamelo pure nel codice aggiornato.